index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Todo App</title>
+  </head>
+  <body>
+    <h1 class="title">투두 리스트</h1>
+    <div class="container">
+      <div>
+        <input autofocus="autofocus" autocomplete="off" type="text" id="taskInput" placeholder="할일을 추가해주세요!"/>
```

Comment:
type, id와 같은 값들이 먼저 나오면 더 깔끔할 것 같아요~
html에 속성을 작성하는 순서도 본인만의 기준을 정해두고 일관되게 가져가면 좋아요!
추가로 autofocus같은 속성은 그대로 사용할 수 있어요~

```suggestion
        <input type="text" id="taskInput" placeholder="할일을 추가해주세요!" autocomplete="off"  autofocus/>
```

Reply:
id가 식별자로 눈에 띠어야하니 앞에 두고 
다음으로는 input 속성에서 구분하기 좋은 type과 placeholder을 두고
나머지를 배치하는 식으로 정하겠습니닷!